pkg/gofr/datasource/pubsub/eventhub/eventhub_test.go

```diff
@@ -107,6 +107,10 @@ func TestConnect_ContainerError(t *testing.T) {
 }
 
 func TestPublish_FailedBatchCreation(t *testing.T) {
+	// TODO: This test is skipped due to long runtime and occasional panic, causing pipeline failures.
+	// It needs modification in the future.
+	t.Skip()
```

Comment:
Add a comment 

```suggestion
	t.Skip("disabled on 2024-12-11 because blah blah, TODO fix this")
```

Here I'm very verbose in the message, it could be simpler